src/App.jsx

```diff
@@ -9,7 +9,7 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <Article {...article} />
```

Comment:
it'd be better to pass values explicitly, like `<Component prop={value} prop2={value2} />`, fix it, because in this way, if you change something in your object it'll be hard to find the issue